detection-rules/spam_campaign_excessive_display_text_with_keywords.yml

```diff
@@ -0,0 +1,17 @@
+name: "Spam: Campaign with excessive display-text and keywords found"
+type: "query"
+description: |
+  Detects affiliate marketing spam where any link contains display-text greater than 3000 chars and specific values found. 
+source: |
+  type.inbound
+  and length(body.links) > 0
+  and (
+    any(body.links, length(.display_text) >3000)
```

Comment:
wow this is crazy. is it 3000+ because of zero-width font / hidden characters? or just a really long hyperlink? (any chance you have a sample we could include as a reference? if not nbd!)

Reply:
yeah absolutely, it's an insane tactic I'll get a sample included.